src/core.py

```diff
@@ -592,7 +600,7 @@ def translate(self, var):
                 for f in dc.fields(TranslatedVarlistEntry) if hasattr(var, f.name)}
             new_name = var.name
         else:
-            fl_entry = self.from_CF(var.standard_name, var.axes_set)
+            fl_entry = self.from_CF(var.standard_name, var.modifier, var.dims.__len__())
```

Comment:
Would've been fine to use len(var.dims) here, since var.dims is just a list, but same difference